src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,76 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) 
+{
+    //if the parameter type is not a valid string throw an error 
+    try
+    {
+      if(typeof text !== "string") 
+        throw `Please enter a valid string`;
```

Comment:
You don't need to wrap this in a try...catch if you're expecting an error. Also, use braces.